packages/graphql/tests/api-v6/schema/invalid-schema/invalid-default-usage-on-list-fields.test.ts

```diff
@@ -0,0 +1,130 @@
+        },
+        { dataType: "[Int]", value: 1.2, errorMsg: "@default.value on Int list fields must be a list of Int values" },
+        {
+            dataType: "[Float]",
+            value: ["stuff"],
+            errorMsg: "@default.value on Float list fields must be a list of Float values",
+        },
+        {
+            dataType: "[DateTime]",
+            value: ["dummy"],
+            errorMsg: "@default.value on DateTime list fields must be a list of DateTime values",
+        },
+    ] as const)(
+        "@default should fail with an invalid $dataType value",
+        async ({ dataType, value: value, errorMsg }) => {
+            const stringValue = typeof value === "string" ? `"${value}"` : value;
+            const fn = async () => {
+                const typeDefs = /* GraphQL */ `
+                    type User @node {
+                        name: ${dataType} @default(value: ${stringValue})
+                    }
+                    extend type User {
+                        anotherField: ${dataType} @default(value: ${stringValue})
+                    }
+                `;
+                const neoSchema = new Neo4jGraphQL({ typeDefs });
+                const schema = await neoSchema.getAuraSchema();
+                raiseOnInvalidSchema(schema);
+            };
+
+            await expect(fn()).rejects.toEqual([new GraphQLError(errorMsg), new GraphQLError(errorMsg)]);
+        }
+    );
+
+    test.each([
+        {
+            dataType: "[ID]",
+            value: ["some-unique-id", "another-unique-id"],
+        },
+        {
+            dataType: "[String]",
+            value: ["dummyValue", "anotherDummyValue"],
+        },
+        {
+            dataType: "[Boolean]",
+            value: [false, true],
+        },
+        { dataType: "[Int]", value: [1, 3] },
+        { dataType: "[Float]", value: [1.2, 1.3] },
+        { dataType: "[DateTime]", value: ["2021-01-01T00:00:00", "2022-01-01T00:00:00"] },
+    ] as const)("@default should not fail with a valid $dataType value", async ({ dataType, value }) => {
+        const fn = async () => {
+            const stringValue = value.map((v) => (typeof v === "string" ? `"${v}"` : v)).join(", ");
+            const typeDefs = /* GraphQL */ `
+                type User @node {
+                    name: ${dataType} @default(value: [${stringValue}])
+                }
+                extend type User {
+                    anotherField: ${dataType} @default(value: [${stringValue}])
+                }
+            `;
+
+            const neoSchema = new Neo4jGraphQL({ typeDefs });
+            const schema = await neoSchema.getAuraSchema();
+            raiseOnInvalidSchema(schema);
+        };
+        await expect(fn()).resolves.not.toThrow();
+    });
+
+    test.todo("add tests for LocalDateTime, Date, Time, LocalTime, Duration when supported");
+    test.todo("@default with custom enum");
+    test.todo("@default with user defined scalar");
```

Comment:
I can't because these are not yet implemented `custom enums and user-defined scalars` while the other types are supported on v6 but were never supported for `@default`. 
If I get the time, I'll probably try to open a PR to extend the support on v5!